.gitignore

```diff
@@ -20,3 +20,6 @@ catkin_ws/*.pyc
 src/cruise_control/scripts/speed_output.txt
 .idea
 
+src/ros_navigation/src/binvox maps/viewvox.exe
```

Comment:
Are these files still used? Looks like old ROS nav stuff

Reply:
Yea, that's not suppose to be there